classes/Attack.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+class Attack
+{
+    private $attacker;
+    private $defender;
+    private $strength;
```

Comment:
I think property strength needs a rename ,because strength can mean a lot of things!

Reply:
If strength is in the attack class would be redundant to say attack strength. If t is a strength property in an attack class is obvious that is attack strength,